pmm-app/src/pmm-add-instance/panel.tsx

```diff
@@ -1,48 +1,54 @@
-import React, { useState } from 'react';
-import { Button } from 'antd';
+import React, { useEffect, useState } from 'react';
 import { createBrowserHistory } from 'history';
-import { Router, Route } from 'react-router-dom';
+import { Router, Route, useLocation } from 'react-router-dom';
+import { Button } from '@grafana/ui';
+import { cx } from 'emotion';
 import AddRemoteInstance from './components/AddRemoteInstance/AddRemoteInstance';
-import DiscoveryPanel from './components/DiscoveryPanel/DiscoveryPanel';
+import Discovery from './components/Discovery/Discovery';
 import { AddInstance } from './components/AddInstance/AddInstance';
-import './panel.scss';
-import '../shared/style.less';
-import '../shared/styles.scss';
+import { getStyles } from './panel.styles';
+import { Messages } from './components/AddRemoteInstance/AddRemoteInstance.messages';
 
+const availableInstanceTypes = ['rds', 'postgresql', 'mysql', 'proxysql', 'mongodb', 'proxysql'];
 const history = createBrowserHistory();
```

Comment:
react-router-dom also feature a `useHistory`, so you may want to consider using it.

Reply:
Thanks, replaced